.storybook/preview.css

```diff
@@ -0,0 +1 @@
+/* Add story preview CSS here  */
```

Comment:
I don't need it's necessary to keep this file.. we can delete it

Reply:
Removed